lighthouse-core/audits/critical-request-chains.js

```diff
@@ -15,7 +15,7 @@ class CriticalRequestChains extends Audit {
   static get meta() {
     return {
       id: 'critical-request-chains',
-      title: 'Critical Request Chains',
+      title: 'Minimize Critical Request Chains',
```

Comment:
the audit is marked `notApplicable` if no requests are made, so everyone who gets the title could in theory minimize their chains. There might not be any benefit to doing that, but that's just a fundamental issue with the audit that needs to be addressed elsewhere :)